FlowCal/transform.py

```diff
@@ -118,6 +128,15 @@ def to_rfi(data, channels=None, max_range=None, amplification_type=None):
         the first element is zero, the amplification type is linear. If
         None, take `amplification_type` from
         ``data.amplification_type(channel)``.
```

Comment:
If you have suggestions to improve readability, I'm all ears. It reads ok to me.

Referencing the `$PnE` sounds more like something that should go in a `Notes` section instead of the description of the argument. I could do this.